src/views/Home/index.vue

```diff
@@ -2,7 +2,7 @@
 <template>
   <div class="home">
     <v-container class="my-5">
-      <h1 class="my-5 display-1 black--text text-uppercase">home</h1>
+      <h1 class="my-5 display-1 font-weight-light">Home</h1>
```

Comment:
Can't change the font here without changing it for the other pages as well. we need to choose which font looks best and apply it to all pages (all caps heavy or what you have now)